### PR TITLE
Feature/waze + Fix event

### PR DESCRIPTION
**Pull request by efratush**
#312 

- Add a link to waze navigation
- (Need to arrange the link with the coordinates after it goes up to code)
- Restore deleted parts from html

### CODE REVIEW

File: app/routers/event.py [hunk 1]
```diff
@@ -109,18 +110,30 @@ async def create_new_event(request: Request,
...
     return RedirectResponse(router.url_path_for('eventview', event_id=event.id)
-                            + f'messages={"---".join(messages)}',
+                            + f'?messages={"---".join(messages)}',
                             status_code=status.HTTP_302_FOUND)
 
 
+def get_waze_link(event: Event) -> str:
+    # if event.latitude and event.longitude:
+    #     coordinates = f"{event.latitude},{event.longitude}"
+    #     return f"https://waze.com/ul?ll={coordinates}&navigate=yes"
```
Comment (PureDreamer) [inline app/routers/event.py:176]: delete?

File: tests/test_event.py [hunk 2]
```diff
@@ -132,6 +132,17 @@ def test_eventview_with_id(event_test_client, session, event):
     response = event_test_client.get(f"/event/{event_id}")
     assert response.ok
     assert b"View Event" in response.content
+    assert b"Some random location" in response.content
+    waze_link = b"https://waze.com/ul?q=Some%20random%20location"
```
Comment (PureDreamer) [inline tests/test_event.py:169]: XD seems good!

File: app/routers/event.py (hunk 1 above)
Comment (efratush) [reply app/routers/event.py:176]: No.  It is based on data waiting to enter the code

**Review by ivarshav — APPROVED** [review]

File: app/routers/event.py [hunk 3]
```diff
@@ -169,6 +170,23 @@ def raise_for_nonexisting_event(event_id: int) -> None:
...
+        Otherwise, waze will look for the address that appears in the location.
+        If there is no address, an empty string will be returned."""
+
+    # if event.latitude and event.longitude:
+    #     coordinates = f"{event.latitude},{event.longitude}"
+    #     return f"https://waze.com/ul?ll={coordinates}&navigate=yes"
+    if event.location:
+        url_location = urllib.parse.quote(event.location)
+        return f"https://waze.com/ul?q={url_location}&navigate=yes"
+    return ""
```
Comment (yammesicka) [inline app/routers/event.py]: Prefer to return early (the "guard" pattern):

```
if not event.location:
    return ""
url_location = urllib.parse.quote(event.location)
return f"https://waze.com/ul?q={url_location}&navigate=yes"
```

Less indented and easier to expand later

File: app/templates/eventview.html [hunk 4]
```diff
@@ -1,4 +1,12 @@
-<link href="{{ url_for('static', path='event/eventview.css') }}" rel="stylesheet">
+{% extends 'base.html' %}
+
+{% block head %}
+    {{ super() }}
+    <title>View Event</title>
```
Comment (yammesicka) [inline app/templates/eventview.html]: It creates 2 `<title>` tags, I think

**Review by yammesicka — CHANGES_REQUESTED** [review]

**Comment by yammesicka** [comment]
Awesome job! I've added 2 small insights.

**Review by yammesicka — APPROVED** [review]

**Pull request merged**